src/main/java/com/waverleysoftware/listemers/SecondTestListener.java

```diff
@@ -0,0 +1,22 @@
+package com.waverleysoftware.listemers;
+
+import org.testng.ISuite;
+import org.testng.ISuiteListener;
+
+import java.util.logging.Logger;
+
+@SuppressWarnings("JavadocType")
+public class SecondTestListener implements ISuiteListener {
+
+    private static final Logger LOG = Logger.getLogger(TestListener.class.getName());
```

Comment:
try replacing common java logger with `@Slf4j` annotation, which is a part of Lombok library

Reply:
I did it in the  **5-lombok-assert-annotation** branch. Please, look at the changes.